ietf-voucher-constrained-sid.txt

```diff
@@ -1,30 +1,30 @@
-           
-      SID Assigned to
---------- -------------------------------------------------- 
-     2467 data /ietf-constrained-voucher:voucher
-     2468 data /ietf-constrained-voucher:voucher/assertion
-     2469 data /ietf-constrained-voucher:voucher/created-on
-     2470 data .../domain-cert-revocation-checks
-     2471 data /ietf-constrained-voucher:voucher/expires-on
-     2472 data /ietf-constrained-voucher:voucher/idevid-issuer
-     2473 data .../last-renewal-date
-     2474 data /ietf-constrained-voucher:voucher/nonce
-     2475 data .../pinned-domain-cert
-     2476 data .../pinned-domain-pubk
-     2477 data .../pinned-domain-pubk-sha256
-     2478 data /ietf-constrained-voucher:voucher/serial-number
-     2451 data /ietf-voucher-constrained:voucher
-     2452 data /ietf-voucher-constrained:voucher/assertion
-     2453 data /ietf-voucher-constrained:voucher/created-on
-     2454 data .../domain-cert-revocation-checks
-     2455 data /ietf-voucher-constrained:voucher/expires-on
-     2456 data /ietf-voucher-constrained:voucher/idevid-issuer
-     2457 data .../last-renewal-date
-     2458 data /ietf-voucher-constrained:voucher/nonce
-     2459 data .../pinned-domain-cert
-     2460 data .../pinned-domain-pubk
-     2461 data .../pinned-domain-pubk-sha256
-     2462 data /ietf-voucher-constrained:voucher/serial-number
-           
- WARNING, obsolete definitions
-           
+           
+      VK Assigned to
+--------- -------------------------------------------------- 
+     2467 data /ietf-constrained-voucher:voucher
+        1 data /ietf-constrained-voucher:voucher/assertion
+        2 data /ietf-constrained-voucher:voucher/created-on

```

Comment:
I thought we were going to use the same values, as if we had used yang-sid, but not depending directly upon it.
Are we using the delta encoding?